README.md

```diff
@@ -15,18 +15,19 @@ uutils coreutils
 -----------------------------------------------
 
 uutils is an attempt at writing universal (as in cross-platform) CLI
-utils in [Rust](http://www.rust-lang.org). This repo is to aggregate the GNU
-coreutils rewrites.
+utilities in [Rust](http://www.rust-lang.org). This repository is intended for
+aggregating the GNU coreutils rewrites.
 
 Why?
 ----
 
-Many GNU, Linux and other utils are pretty awesome, and obviously
+Many GNU, Linux and other utilities are useful, and obviously
 [some](http://gnuwin32.sourceforge.net) [effort](http://unxutils.sourceforge.net)
 has been spent in the past to port them to Windows. However, those projects
-are either old, abandoned, hosted on CVS, written in platform-specific C, etc.
+are either old, abandoned, hosted on CVS, written in platform-specific C, or
```

Comment:
I’d probably change `old, abandoned` to `old/abandoned`.  Also I feel like we should clarify that `hosted on CVS` is a negative as it makes it more difficult for new contributors.